frank-doc-frontend/src/app/views/properties/properties.component.html

```diff
@@ -24,7 +24,11 @@
     }
   </div>
   <div class="page">
-    <h2 class="page-title">AppConstants</h2>
+    <h2 class="page-title">Properties</h2>
```

Comment:
Are the frank developers going to understand that these are the AppConstants?

Reply:
AppConstants is the technical term we're using they mostly use the word properties or variables.